src/store/plugins/observables.js

```diff
@@ -19,6 +20,31 @@ export default (store) => {
       refCountDelay(1000),
     );
 
+  const balance$ = sdk$.pipe(
+    switchMap((sdk) => timer(0, 3000).pipe(map(() => sdk))),
+    switchMap((sdk) =>
+      sdk
+        ? sdk
+            .balance(store.state.account.publicKey)
+            .catch((error) => {
+              if (!isNotFoundError(error)) {
+                handleUnknownError(error);
+              }
+              return 0;
+            })
+            .then((result) => {
+              const balance = convertToAE(result);
+              if (balance !== store.state.balance) {
+                store.commit('updateBalance', balance);
+              }
+              return balance;
+            })
+        : Promise.resolve(store.state.balance),
+    ),
+    multicast(new BehaviorSubject(store.state.balance)),
```

Comment:
The arguments to don't store the balance in store:
- developers may use the balance from the store not aware that it won't be updated without a subscription to the corresponding observable variable
- balance polling emits wasteful mutations that can complicate debugging
- the cached balance not needed to be handled the same careful (don't need to maintain in migrations) as other state fields persisted in local storage

Proposing to sync it with local storage explicitly, using a different key. For example, to get the balance by:
```js
JSON.parse(localStorage['rxjs']).balance
```
and to set it by:
```js
localStorage['rxjs'] = JSON.stringify({ ...JSON.parse(localStorage['rxjs']), balance })
```
even better would be to write a helper functions for that